examples/cd_context_20newsgroup.ipynb

```diff
@@ -0,0 +1,3186 @@
+{
```

Comment:
<p>I don't understand this. Why do we wish to allow the distribution of embeddings to change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>